drivers/gpu/drm/virtio/virtgpu_display.c

```diff
@@ -118,6 +118,14 @@ static void virtio_gpu_crtc_atomic_disable(struct drm_crtc *crtc,
 	struct virtio_gpu_device *vgdev = dev->dev_private;
 	struct virtio_gpu_output *output = drm_crtc_to_virtio_gpu_output(crtc);
 	const unsigned pipe = drm_crtc_index(crtc);
+	struct drm_vblank_crtc *vblank = &dev->vblank[pipe];
+	int ret = 0;
+
+	if (vgdev->cache_event[pipe]) {
+		ret = wait_for_completion_interruptible_timeout(&vgdev->notify[pipe], nsecs_to_jiffies(vblank->framedur_ns));
```

Comment:
need separate it into two lines, linux kernel has restriction for the length of line